README.md

```diff
@@ -1,6 +1,6 @@
 # Deprecated!
 
-As of Feb 11th 2020, request is fully deprecated. No new changes are expected to land. In fact, none have landed for some time.
+As of Feb 11th 2020, request is fully deprecated. No new changes are expected land. In fact, none have landed for some time.
```

Comment:
land (something)
To successfully acquire something, such as a job or piece of information.
The economy is still in terrible shape—I haven't been able **to** land a job for months.
Tom *landed* a really juicy story about the senator's ex-wife.
source: [thefreedictionary.com/land+something](https://idioms.thefreedictionary.com/land+something)

> Ha! Just in time for the history books, I'm now officially a request contributor 😈 Love when my plan comes together
_Originally posted by @gr2m in https://github.com/request/request/pull/3268#issuecomment-584920736_

**We want @gr2m  plan(e) ✈️  to land successfully as it comes together so please remove this PR in order for #3268 to be the one final answer... And nonetheless « No new changes are expected *to* land. In fact, none have *landed* for some time »**